src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
-body {
+$first-color: #060b35;
+$second-color: #616070;
+$button-color: #00acdc;
+
+html,
+body,
+div,
+h4,
+p,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  min-width: 200px;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  font-family: Roboto, Helvetica, sans-serif;
```

Comment:
You have a fixed width, you don't need min-width in this case.
The main font-family should apply to the html or body tag.